Tests/store_packs.py

```diff
@@ -0,0 +1,512 @@
+        pack_metadata['contentItems'] = content_items_data
+        pack_metadata["contentItemTypes"] = list(content_items_data.keys())
+        # todo collect all integrations display name
+        pack_metadata["integrations"] = []
+        pack_metadata["useCases"] = input_to_list(user_metadata.get('useCases'))
+        pack_metadata["keywords"] = input_to_list(user_metadata.get('keywords'))
+        pack_metadata["dependencies"] = {}  # TODO: build dependencies tree
+
+        return pack_metadata
+
+    def zip_pack(self):
+        """Zips pack folder and excludes not wanted directories.
+
+        Returns:
+            str: full path to created pack zip.
+
+        """
+        zip_pack_path = f"{self._pack_path}.zip"
+
+        with ZipFile(zip_pack_path, 'w', ZIP_DEFLATED) as pack_zip:
+            for root, dirs, files in os.walk(self._pack_path, topdown=True):
+                dirs[:] = [d for d in dirs if d not in Pack.EXCLUDE_DIRECTORIES]
+
+                for f in files:
+                    full_file_path = os.path.join(root, f)
+                    relative_file_path = os.path.relpath(full_file_path, self._pack_path)
+                    pack_zip.write(filename=full_file_path, arcname=relative_file_path)
+
+        print_color(f"Finished zipping {self._pack_name} pack.", LOG_COLORS.GREEN)
+
+        return zip_pack_path
+
+    def upload_to_storage(self, zip_pack_path, latest_version, storage_bucket):
+        """ Manages the upload of pack zip artifact to correct path in cloud storage.
+        The zip pack will be uploaded to following path: /content/packs/pack_name/pack_latest_version.
+        In case that zip pack artifact already exist at constructed path, the upload will be skipped.
+
+        Args:
+            zip_pack_path (str): full path to pack zip artifact.
+            latest_version (str): pack latest version.
+            storage_bucket (google.cloud.storage.bucket.Bucket): google cloud storage bucket.
+
+        Returns:
+            bool: True is pack was successfully uploaded. False in case that packs already exists at the bucket.
+
+        """
+        version_pack_path = os.path.join(STORAGE_BASE_PATH, self._pack_name, latest_version)
+        existing_files = [f.name for f in storage_bucket.list_blobs(prefix=version_pack_path)]
+
+        if existing_files:
+            print_warning(f"The following packs already exist at storage: {', '.join(existing_files)}")
+            print_warning(f"Skipping step of uploading {self._pack_name}.zip to storage.")
+            return False
+
+        pack_full_path = f"{version_pack_path}/{self._pack_name}.zip"
+        blob = storage_bucket.blob(pack_full_path)
+
+        with open(zip_pack_path, "rb") as pack_zip:
+            blob.upload_from_file(pack_zip)
+        print_color(f"Uploaded {self._pack_name} pack to {pack_full_path} path.", LOG_COLORS.GREEN)
+
+        return True
+
+    def format_metadata(self):
```

Comment:
not sure I understand this func, Need to make sure that our internal metadata name was changed.

Reply:
Changed internal name